app/models/cart.client.ts

```diff
@@ -0,0 +1,189 @@
+    query: queries.cartLinesUpdate,
+    variables: {
+      cartId,
+      id,
+      merchandiseId,
+      quantity,
+    },
+  };
+  const response = await postToShopify(params);
+  return response;
+};
+
+//GraphQl queries object
+const queries = {
+  getCart: `query getCart($id: ID!) {
+    cart(id: $id) {
+      estimatedCost {
+        subtotalAmount {
+          amount
+          currencyCode
+        }
+      }
+      lines(first: 100) {
+        edges {
+          node {
+            id
+            quantity
+            merchandise {
+              ... on ProductVariant {
+                id
+                image {
+                  altText
+                  src
+                }
+                title
+                product {
+                  id
+                  description
+                  title
+                  priceRange {
+                    minVariantPrice {
+                      amount
+                      currencyCode
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }`,
+  createCart: `mutation createCart {
+    cartCreate {
+      cart {
+        id
+        checkoutUrl
+      }
+    }
+  }
+`,
+  addItemToCart: `mutation MyMutation($cartId: ID!, $lines: [CartLineInput!] = {merchandiseId: "Z2lkOi8vc2hvcGlmeS9Qcm9kdWN0VmFyaWFudC8zNTU1NjQwNjEwMDEzMQ==", quantity: 1}) {
+    cartLinesAdd(cartId: $cartId, lines: $lines) {
+      cart {
+        id
+        lines(first: 100) {
+          edges {
+            node {
+              id
+              quantity
+              merchandise {
+                ... on ProductVariant {
+                  id
+                  priceV2 {
+                    amount
+                  }
+                  title
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`,
+  removeItemFromCart: `mutation removeItemFromCart($cartId: ID!, $lineIds: [ID!]!) {
+    cartLinesRemove(cartId: $cartId, lineIds: $lineIds) {
+      cart {
+        id
+        lines(first: 10) {
+          edges {
+            node {
+              id
+              quantity
+              merchandise {
+                ... on ProductVariant {
+                  id
+                  title
+                  priceV2 {
+                    amount
+                    currencyCode
+                  }
+                  product {
+                    title
+                    handle
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  `,
+  cartLinesUpdate: `mutation MyMutation($cartId: ID = "Z2lkOi8vc2hvcGlmeS9DYXJ0L2Y5MWM1MjM1ZDdkY2Q5MGYyMTVkNTQ1NjBkYTM5YjZi", $id: ID = "Z2lkOi8vc2hvcGlmeS9DYXJ0TGluZS9jYzQ1MmNhMi0zM2FiLTQwZWYtYThjYi02YTVkYmZjMTkyM2E/Y2FydD1mOTFjNTIzNWQ3ZGNkOTBmMjE1ZDU0NTYwZGEzOWI2Yg==", $merchandiseId: ID = "Z2lkOi8vc2hvcGlmeS9Qcm9kdWN0VmFyaWFudC8zNTU1NjUyMTcwNTYzNQ==", $quantity: Int = 5) {
```

Comment:
why is this in such a mess? lol

Reply:
I took them off.

Those are default values in cases when we send "buggy" requests, 
basically, if you see this cart details you know "something went wrong" 
it's instead of making the whole app collapse.